package.json

```diff
@@ -31,6 +32,7 @@
     "babel-loader": "^6.2.7",
     "babel-polyfill": "^6.16.0",
     "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.16.0"
+    "babel-preset-react": "^6.16.0",
+    "babel-preset-stage-0": "^6.24.1"
```

Comment:
shouldn't this be version 6.16.0 like the rest of the babel imports?

Reply:
Also don't care if the order is re-shuffled must be something you run locally, but we can roll with this